content/en/05-reference/03-plugins/01-plugin-system.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Pixie Plugin System"
+metaTitle: "Reference | Plugins | Pixie Plugin System "
+metaDescription: "Pixie's Plugin System"
+order: 1 
+---
+
+Pixie's core functionality focuses on providing a real-time debugging platform. Integrate with other tools using the Pixie Plugin system to enable capabilities like longterm data retention or alerting.
+
+- *Longterm Data Retention*: Pixie does not guarantee data storage for over 24 hours. Leverage an external datastore for longterm data retention by sending Pixie data in the [OpenTelemetry](https://opentelemetry.io/) format. Future support will be added for querying longterm data from within the Pixie UI, allowing you to use PxL and Pixie's versatile live views.
+- *Alerts* (Coming Soon!): Power alerts using Pixie's rich dataset, all configurable from within Pixie's UI.
+
+## Enabling a Plugin
+
+Plugins can be configured from within the Admin UI. 
+
+1. Open the Live UI and navigate to the Plugins tab on the Admin page (`/admin/plugins`).
+2. Find the plugin you wish to enable and click the toggle to enable/disable the plugin.
+3. Expand the plugin row to specify the necessary configuration values. The required set of fields differs per plugin provider, and usually contains information such as API keys which are used for authentication.
```

Comment:
Link to /reference/admin/api-keys.

Reply:
This is the API key for the plugin provider, not ours.